src/actions/MastodonAddVariablesAction.C

```diff
@@ -22,47 +22,47 @@
 
 template <>
 InputParameters
-validParams<MastodonAddVariableAction>()
+validParams<MastodonAddVariablesAction>()
 {
   InputParameters params = validParams<Action>();
   params.addClassDescription("Create displacement Variables, and velocity, acceleration, stress "
                              "and strain AuxVariables automatically.");
 
   params.addParam<bool>(
-      "add_variables", true, "false, if displacement Variables are created manually.");
+      "variables", true, "false, if displacement Variables are created manually.");
```

Comment:
Is there a reason why there are two separate actions - one for setting up variables and the other for setting kernels/auxkernels? 

What happens when variables/auxvariables are set by the user using different names than the ones used in kernel action? Also does the order in which these actions are executed make a difference? i.e., if I add `inertia_kernels = true` before `variables=true`, does it generate an error saying the variables are not set up yet?

Reply:
@sveerara I thought it would be better to organize the code that way, but I do get your point. The main goal of creating these actions (and the Mastodon block) is to find the most widely used parameters and make them default in this block, and the user doesn't have to worry about them. The simulation will fail if different variable names are used for variables and kernels, but in most cases the names will be the same. I need to think some more about the right design for this action. Do you think it is better to create one big action for variables and kernels and create error checks for names, etc?